doc/lua_api.md

```diff
@@ -6378,6 +6380,12 @@ Environment access
 * `core.get_day_count()`: returns number days elapsed since world was
   created.
     * Time changes are accounted for.
+* `core.get_target_dtime()`: returns the target server step `dtime` in seconds.
+    * This is a lower bound, i.e. server steps may not be shorter than this,
+      but they are often longer.
```

Comment:
this contradicts the "This is usually true: `dtime >= core.get_target_dtime()`" and I don't think we want to (or can) guarantee it